frontend-next/templates/first/config.js

```diff
@@ -1,13 +1,40 @@
 const config = {
-    sponsor: {
-        imageUrl: 'https://raw.githubusercontent.com/atherdon/newsletters/master/archive/logos/thematic/2020/September/flatfile-logo-black-small-vertical.png',
-        link: 'https://bit.ly/30AWyC4',
-        text: 'Ship the data importer you always dreamed of'
-    },
-    mainTitle: 'Can We Make Data Tidy?',
-    paragraphFirst: `Imagine: You are going to sit down with a newly-fetched data set,
+  sponsor: {
+    imageUrl: 'https://raw.githubusercontent.com/atherdon/newsletters/master/archive/logos/thematic/2020/September/flatfile-logo-black-small-vertical.png',
+    link: 'https://bit.ly/30AWyC4',
+    text: 'Ship the data importer you always dreamed of',
+  },
+  mainTitle: 'Can We Make Data Tidy?',
+  paragraphFirst: `Imagine: You are going to sit down with a newly-fetched data set,
     excited about the insights it will bring you and then you find out it is no use.
     If you’ve been there, then you know for sure what an untidy dataset is.`,
-}
+  memImageLink: 'https://raw.githubusercontent.com/atherdon/newsletters/master/archive/img/memes/6.jpg',
+  paragraphSecond: `A statistician from New Zealand once said: Tidy datasets are all alike, but every messy dataset
+    is messy in its own way. Indeed, as data may come in various forms and shapes, sometimes we are
+    inundated with it. As a result, our data science team becomes shortsighted and oops..
+    disillusioned by mountains of unworkable data. The only way data specialists can facilitate analysis is
+    by keeping data clean and organized.`,
+  heading: 'What is tidy data?',
+  paragraphThird: `Essentially, tidy data is a term coined by Hadley Wickham in his Tidy Data paper (remember that
+        statistician from NZ?).
+        He defined tidy data as data that is neatly organized and all set for analysis. This way of organizing
+        allows you to easily produce charts,
+        diagrams, and summary statistics. As it often happens, not all data comes out of the database clean,
+        therefore cleansing it is essential
+        to efficiently analyze it.`,
+  memImageLink2: 'https://raw.githubusercontent.com/atherdon/newsletters/master/archive/img/memes/1.png',
+  paragraphFourth: 'Without further ado, let us break down the principles that allow you keep your data nice and clean.',
+  heading2: 'Tidy Data Principles',
+  heading3: '1. Each row is an observational unit.',
```

Comment:
I don't think it would work as it is right now.
config should contain only basic stuff. content will be moved separately.